scripts/backup.py

```diff
@@ -81,15 +81,17 @@ def backup_roles():
 
 
 def backup():
+    contents = dict(
+        users=backup_users(),
+        groups=backup_groups(),
+        roles=backup_roles())
     with open('backup.json', 'w') as fp:
```

Comment:
Would it be worthwhile to include the current date/time in the filename?  This would prevent the backups from being overwritten too.

Otherwise LGTM.

Reply:
adding timedate